src/assets/scss/variables/_color.variables.module.scss

```diff
@@ -46,8 +46,12 @@ $tb-transparent: transparent;
 
 // background
 $tb-black-background: #2d2d2d;
+$tb-black-background-shadow: #2d2d2dcc;
```

Comment:
It may be better if we use `5d5d5d`, I think it's the same color and it is easy to understand 😅

Reply:
it should be 2d2d2d then according to the design, I will remove the shadow